ko/admin/config.rst

```diff
@@ -2271,7 +2271,9 @@ HA 관련 파라미터
 
 **vacuum_ovfp_check_duration**
 
- **vacuum_ovfp_check_duration**\는 **vacuum_ovfp_check_threshold**\ 설정에 의해 수집된 정보를 보관할 기간을 설정한다. 설정한 기간 내에 업데이트되지 않은 정보는 삭제된다. 입력값의 단위를 생략하면 밀리초이며 기본값은 31일에 해당하는 744 시간이다. 설정할 때 단위로 ms/s/min/h를 사용할 수 있다.
+ **vacuum_ovfp_check_duration**\는 vacuum 쓰레드에 의해 수집된 인덱스 오버플로우 수와 그 관련 정보를 유지하는 기간을 지정한다. 설정한 기간 내에 업데이트되지 않은 정보는 제거된다. 
+     
+     값의 단위는 분이다.
```

Comment:
please remove the line, ㅠecause it is combined with the preceding sentence.

Reply:
I will fix it.